templates/rsyslog.conf.erb

```diff
@@ -39,6 +47,16 @@ $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
 # not useful and an extreme performance hit
 #$ActionFileEnableSync on
 <% end -%>
+
+<% if @mod_imjournal.to_s == 'true' -%>
+# Turn off message reception via local log socket;
```

Comment:
Could these lines be in the placed above (line 36) with the other new section or does it have to be here?